docs/Conditions.md

```diff
@@ -8,8 +8,8 @@ Name | Type | Description | Notes
 **actions** | **list[list[str]]** | True if any of the specified lists of actions have been taken; multiple action ID lists have \&quot;or\&quot; semantics; multiple action IDs within a list have \&quot;and\&quot; semantics | [optional] 
 **probes** | **list[str]** | True if the specified list of probe_ids have been answered | [optional] 
 **probe_responses** | **list[str]** | True if the specified list of probe responses (choice) have been sent | [optional] 
-**character_vitals** | [**list[ConditionsCharacterVitals]**](ConditionsCharacterVitals.md) | True if the specified list of vitals values have been met for the specified character_id | [optional] 
-**supplies** | [**list[Supplies]**](Supplies.md) | True if there are at least the specified quantity of the specified supply types remaining | [optional] 
+**character_vitals** | [**list[ConditionsCharacterVitals]**](ConditionsCharacterVitals.md) | True if the any of the specified collection of vital values have been met for the specified character_id | [optional] 
```

Comment:
typo: "the any of the"

Reply:
Thanks-- will fix and regenerate models in both client and server.  Would you be opposed to me fixing this in ITM-386 (which is branched off of ITM-409) if this ends up being the only change to 409?  It avoids having to merge in a multi-file commit from ITM-409 when it can easily be fixed in ITM-386...plus I know I'll already have another commit in ITM-386 to remove the test file.

Reply:
Sounds good to me